diff --git a/Containerfile b/Containerfile
index c92028d..01e405b 100644
--- a/Containerfile
+++ b/Containerfile
@@ -8,10 +8,10 @@ ADD tunables.conf /usr/lib/sysctl.d/tunables.conf
 RUN \
 set -x && \
 # Memory allocator
-curl --create-dirs -Lo /usr/lib64/libmimalloc-secure.so https://github.com/charles8191/mimalloc-secure/raw/refs/heads/main/libmimalloc-secure.so && \
-chmod +x /usr/lib64/libmimalloc-secure.so && \
-echo "/usr/lib64/libmimalloc-secure.so" > /etc/ld.so.preload && \
-echo "/usr/lib64/libmimalloc-secure.so" > /usr/etc/ld.so.preload && \
+curl --create-dirs -Lo /usr/lib64/libSlimGuard.so https://github.com/charles8191/SlimGuard/raw/refs/heads/main/libSlimGuard.so && \
+chmod +x /usr/lib64/libSlimGuard.so && \
+echo "/usr/lib64/libSlimGuard.so" > /etc/ld.so.preload && \
+echo "/usr/lib64/libSlimGuard.so" > /usr/etc/ld.so.preload && \
 # Branding
 sed -i 's,rockylinux.org,github.com/charles8191/netherite,g' /usr/lib/os-release && \
 sed -i 's,Rocky Linux,Netherite,g' /usr/lib/os-release && \
diff --git a/README.md b/README.md
index 5ffd478..60a7047 100644
--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ Netherite is a secure & private operating system based on [Calcite](https://gith
 
 ## Features
 
-- [mimalloc](https://github.com/microsoft/mimalloc) (secure mode) using [weekly mimalloc-secure binaries](https://github.com/charles8191/mimalloc-secure)
+- [SlimGuard allocator](https://github.com/ssrg-vt/SlimGuard) using [weekly binaries](https://github.com/charles8191/SlimGuard)
 - Some remediations from ANSSI-BP-028 Minimal
 - [LibreWolf](https://librewolf.net/) instead of Firefox
 - Custom chrony config
